src/credentials/vue/authForms/CredentialsForm.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+    <div class="auth-form container-background border-common" id="credentials-form">
+        <FormTitle :isConnected="isConnected">IAM Credentials</FormTitle>
+
+        <div class="form-section">
+            <label class="small-description">Credentials will be added to the appropriate `.aws/` files.</label>
```

Comment:
Looks like a relative path. "~/" is understood to mean home even on Windows. Could also consider showing the resolved path here.

```suggestion
            <label class="small-description">Credentials will be added to the appropriate `~/.aws/` files.</label>
```